en/docs/develop/setting-up-user-portal-in-a-dev-environment.md

```diff
@@ -0,0 +1,101 @@
+# Setting-Up User Portal in a Development Environment
+
+In [WSO2 Identity Server 5.10.0](https://wso2.com/identity-and-access-management/) the user portal is available under
+ the URL `http(s)://<SERVER_HOST>/t/<TENANT_DOMAIN>/user-portal`. If you are planning on setting up the repo in a
+ development environment follow the steps given below.
+
+!!! note "Before you begin"
+
+    1. Install [node](https://nodejs.org/en/download/) if you haven’t already(npm is already bundled with node).
+    2. Install [maven](https://maven.apache.org/download.cgi) (Needed to run mvn commands).
+    3. A running Identity Server v 5.10. (If you want to build from source follow the instructions listed [here]
+    (https://github.com/wso2/product-is)).
+
+
+### Step 1: Configuring Identity Server
+
+!!! tip
+    Default dev origin, hostname and port set for the User Portal is https://localhost:9000. So the following
+    configurations will have to change accordingly if you change the port or any other configurations.
```

Comment:
Note that the default dev origin, hostname, and port that is set for the user portal is `https://localhost:9000`. Change the following configurations accordingly if you have changed this port or any other configuration.